api/src/pcapi/infrastructure/worker_container.py

```diff
@@ -13,8 +12,6 @@
 venue_identifier_repository = VenueWithBasicInformationSQLRepository()
 
 # Usecases
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** Usecase au singulier maintenant du coup ?‏

Reply:
Je compte bien nettoyer tout cela plus tard, surtout !